### DISCUSSION TITLE
colorize image

### DISCUSSION THREAD

**Discussion by alexlezam** [ImageMagick/ImageMagick#5216] [Consulting]
Good day.
Just new to using imagemagick.
What happens is that I need to know how to color or change the color of an image. They are white images.
YES I would like to know what is the method to change the color of the image.
the image is clothes, so it does not have a defined shape

**Comment by alexlezam** [comment]
P.S. i'm using c#

**Comment by fmw42** [comment]
You will need to make a mask of your clothing. Then you would change the color of the whole image to the desired color for the clothing. (See the C# equivalent of -modulate). Then you would composite the new image with the original using the mask to decide where to use the new image and where to use the original.  Sorry, I do not program C#. I only work in command line mode.

**Reply by alexlezam** [reply]
thanks.
